### PR TITLE
enable response file for ctors list parameter

### PR DESCRIPTION
**Pull request by coldfire913**
for some large systems, the ctors list can easily exceed the windows limitation

### CODE REVIEW

File: src/tools/wasm-ctor-eval.cpp [hunk 1]
```diff
@@ -369,7 +369,10 @@ int main(int argc, const char* argv[]) {
       .add("--ctors", "-c", "Comma-separated list of global constructor functions to evaluate",
            Options::Arguments::One,
            [&](Options* o, const std::string& argument) {
-             ctorsString = argument;
+               if (!argument.empty() && argument[0] == '@')
```
Comment (kripken) [inline src/tools/wasm-ctor-eval.cpp]: indentation needs some fixing here (2 spaces), and also need `{`, `}` around the if and body.

a larger issue is that we should probably make a utility method in `src/support/file.h` for expanding a response file. perhaps we should even do this by default? or have an option on Options that they have this feature or not?

Comment (coldfire913) [inline src/tools/wasm-ctor-eval.cpp]: thanks for the comment. the indentation is caused by IDE settings required by our company and I have changed to fit the code standards on your side.  
The response file is the default option for our team here as we have thousands of global ctors in the system...  and we prefer to have another PR for the option work.

File: src/tools/wasm-ctor-eval.cpp [hunk 2]
```diff
@@ -369,7 +369,12 @@ int main(int argc, const char* argv[]) {
       .add("--ctors", "-c", "Comma-separated list of global constructor functions to evaluate",
            Options::Arguments::One,
            [&](Options* o, const std::string& argument) {
-             ctorsString = argument;
+             if (!argument.empty() && argument[0] == '@'){
```
Comment (kripken) [inline src/tools/wasm-ctor-eval.cpp]: I meant to put this logic into the utility function `read_response_file`. That way it's easy for other options to support response files too.

File: src/tools/wasm-ctor-eval.cpp [hunk 3]
```diff
@@ -369,7 +369,12 @@ int main(int argc, const char* argv[]) {
...
+               ctorsString = read_response_file(argument.substr(1), options.debug ? Flags::Debug : Flags::Release);
+             }
+             else{
```
Comment (kripken) [inline src/tools/wasm-ctor-eval.cpp]: proper else notation should be `} else {`

**Pull request closed without merging**